src/Avalonia.Diagnostics/Diagnostics/Views/TreePageView.xaml.cs

```diff
@@ -139,9 +139,15 @@ private void InitializeComponent()
             AvaloniaXamlLoader.Load(this);
         }
 
-        private void OnClipboardCopyRequested(object? sender, string e)
+        private void OnClipboardCopyRequested(object? sender, (string Format, string Value) arg)
         {
-            TopLevel.GetTopLevel(this)?.Clipboard?.SetTextAsync(e);
+            if (TopLevel.GetTopLevel(this)?.Clipboard is { } clipboard)
+            {
+                var @do = new DataObject();
+                @do.Set(DataFormats.Text, arg.Value);
+                @do.Set(arg.Format, arg.Value);
```

Comment:
I would assume that you need to keep clean selector without namespaces in Text format, while adding richer format for our IDE plugins. But here you add the same value for both formats.
Am I wrong?

Reply:
Yes, it's the same at the moment. I also thought about creating a richer format (json) but I don't want to add dependencies. Che cosa suggerisci?

Reply:
By "rich" I didn't mean json specifically.

From my understanding, normal Text format is expected to provide basic but usable information:
```
NavigationView#NavView.Store /template/ ScrollViewer#FooterItemsScrollViewer
```

NavigationView namespace is ignored, as we don't know that information there, but it still is a valid syntax.

And for special DevTools format we can pass something like:
```
{FluentAvalonia:FluentAvalonia.UI.Controls}NavigationView#NavView.Store /template/ ScrollViewer#FooterItemsScrollViewer
```
Which includes assembly name and namespace for non-Avalonia types (or all types). So IDE can use this information to quickly find that assembly and type, and update XAML with this information.

Reply:
I was thinking of something like this
```json
{  
  "namespace": "FluentAvalonia:FluentAvalonia.UI.Controls",
  "element": "FluentAvalonia:FluentAvalonia.UI.Controls",
  "name": "NavView",
  "classes": ["Store"],
  "pseudoclass": null,
  "template": {
    "namespace": "Avalonia.Controls",
    "element": "ScrollViewer",
    "name" : "FooterItemsScrollViewer"
  },
  "descendants": null
}
```

I like your idea.